### DISCUSSION TITLE
ARMv7 Docker Version

### DISCUSSION THREAD

**Discussion by kyjus25** [directus/directus#2770] [Archive]
I'd like to run this from a Raspberry Pi. I see that Dockerhub only has versions for amd64, is there a Dockerfile anywhere that would allow me to build this for ARM based CPUs?

**Comment by rijkvanzanten** [comment]
I don't think we have one readily available yet. 

@WoLfulus is this something worth looking into? With Apple moving to ARM based CPUs later this year and Microsoft's Surface X already out in the world it might be a good thing to have ARM env support ootb?
